util/topgen-fusesoc.py

```diff
@@ -154,6 +159,10 @@ def main():
                         'rtl/autogen/%s_rnd_cnst_pkg.sv' % topname,
                         'rtl/autogen/%s_pkg.sv' % topname,
                         'rtl/autogen/%s.sv' % topname,
+                        # TODO: this is not ideal. we should extract
+                        # this info from the target configuration and
+                        # possibly generate separate core files for this.
+                        'rtl/autogen/%s_cw305.sv' % topname,
```

Comment:
As mentioned in the comment, this is not ideal, but does the job for now.
We should come up with a cleaner solution.